app/controllers/new_user/dossiers_controller.rb

```diff
@@ -31,8 +31,43 @@ def update_identite
       end
     end
 
+    def edit
+      @dossier = current_user.dossiers.includes(champs: :type_de_champ).find(params[:id])
+    end
+
+    def update
+      @dossier = current_user.dossiers.includes(champs: :type_de_champ).find(params[:id])
+
+      errors = PiecesJustificativesService.upload!(@dossier, current_user, params)
+
+      if !@dossier.update(champs_params)
+        errors += @dossier.errors.full_messages
```

Comment:
Ça marche bien avec les nouvelles PJ ?

Reply:
Maintenant oui :) ! Par contre, je n'ai pas réussi à faire les tests end to end pour l'instant.